dependencies.py

```diff
@@ -0,0 +1,13 @@
+# from sqlalchemy.orm import Session
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        await db.close()
```

Comment:
Ensure that `await db.close()` is correctly closing the session to prevent any potential resource leaks. This is important for maintaining efficient database connections in an asynchronous application.